src/App.tsx

```diff
@@ -1,18 +1,138 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Routes,
+  Route,
+  useLocation,
+  Navigate,
+} from 'react-router-dom';
 import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
-
+import { HomePage } from './components/HomePage';
+import { PageNotFound } from './components/PageNotFound';
+import { Person, NewPerson } from './types';
+import { getPeople } from './api';
+import { FilterType } from './types/enum';
+import { Loader } from './components/Loader';
 import './App.scss';
 
-export const App = () => {
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [newPeople, setNewPeople] = useState<NewPerson[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [sexFilter, setSexFilter] = useState(FilterType.All);
+  const [query, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const location = useLocation();
+
+  const isHomePage = location.pathname.endsWith('/home');
+
+  const sexFilterHandler = (value: FilterType) => {
+    setSexFilter(value);
+  };
+
+  const deleteQuery = () => {
+    setQuery('');
+    localStorage.setItem('query', '');
+  };
+
+  const fetchPeople = async () => {
+    try {
+      const fetchedData = await getPeople();
+
+      setLoading(false);
+      setPeople(fetchedData);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('Error occurred:', error);
```

Comment:
Don't disable eslint. Remove all console.log. You can show the error message instead to let the user know that an error has occurred